dashboard/static/src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title><%= htmlWebpackPlugin.options.title %></title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css">
```

Comment:
Is there a reason why we would or wouldn't want to use `package.json` exclusively (for all libs)?

Reply:
Hi, ivan, I actually thought about that.  I will try to do that when I have more time.  I also am not sure if bundle size will become a concern...I will do more research on bundle size management before falling back to using package.json for all libs.

Reply:
@achen2401 when you say you'll try when you have more time, I assume that means beyond this PR, as in we should merge and you'll address later?

Reply:
yes, Paul, sorry I wasn't clear.  I am need to switch gear to work on Fenway project.  I will come back to address this afterwards.